src/app.rs

```diff
@@ -389,6 +394,7 @@ impl FinishedApplication {
         }
     }
 
+    #[must_use]
```

Comment:
Were these also related to Windows errors? I'm mostly curious what would require these but not any other code changes.

Reply:
Oh, oops, I had put those in to ensure the library usage was making use of the resulting values, and then left them in the subsequent fixes. They are unrelated but should have been there from the start.